sympy/utilities/tests/test_codegen_julia.py

```diff
@@ -488,7 +488,7 @@ def test_jl_loops():
                       header=False, empty=False)
     source = result[1]
     expected = (
-        'function mat_vec_mult(y, A, m, n, x)\n'
+        'function mat_vec_mult(A, m, n, x, y)\n'
```

Comment:
This looks like it could be a breaking change, no?

Reply:
it's only in Julia where the outputs are at the beginning. I don't know why this choice... It doesn't make sens to me.

But I am a newbie in Julia.